Packs/CommonPlaybooks/TestPlaybooks/playbook-File_Enrichment_-_Generic_v2_-_Test.yml

```diff
@@ -2,7 +2,7 @@ id: File Enrichment - Generic v2 - Test
 version: -1
 fromversion: 5.0.0
 name: File Enrichment - Generic v2 - Test
-description: A test for the File Enrichment - Generic v2 playbook.
+description: A test for the File Enrichment - Generic v2 playbook. with and instance
```

Comment:
?

Reply:
Ooops this is actually a test, will revert after the check. sorry forgot to remove you for review